src/sentry/models/groupsubscription.py

```diff
@@ -88,19 +88,21 @@ def get_participants(self, group):
 
         # Find users which by default do not subscribe.
         participating_only = set(
-            UserOption.objects.filter(
+            uo.user_id for uo in UserOption.objects.filter(
```

Comment:
@tkaemming kind of annoying, but i think still worth it for sane defaults on the model